templates/etc/postgresql/9.6/main/pg_hba.conf.j2

```diff
@@ -1,17 +1,8 @@
 # TYPE  DATABASE        USER            ADDRESS                 METHOD
-
-# Database administrative login by Unix domain socket
 local   all             postgres                                peer
-
-# "local" is for Unix domain socket connections only
 local   all             all                                     peer
-
-# IPv4 local connections:
-host    all             all             127.0.0.1/32            md5
-
-# IPv6 local connections:
-host    all             all             ::1/128                 md5
-
+host    all             all             0.0.0.0/0               md5
```

Comment:
I "had" to add this, otherwise, only users from `localhost` can connect.

So maybe I should add a var? I can restore your defaults too with it.

```
postgresql_config_hba:
  - type: host
     database: all
     user: all
     address: 0.0.0.0/0
     method: md5
  - type: host
     database: all
     user: all
     address: ::/0
     method: md5
```

Thoughts?

Reply:
I think this `postgresql_config_hba` var would be good enough as you have shown here.